libfs/fs.go

```diff
@@ -276,7 +311,42 @@ func (fs *FS) ReadDir(p string) (fis []os.FileInfo, err error) {
 }
 
 // MkdirAll implements the billy.Filesystem interface for FS.
-func (fs *FS) MkdirAll(filename string, perm os.FileMode) error {
+func (fs *FS) MkdirAll(filename string, perm os.FileMode) (err error) {
+	fs.log.CDebugf(fs.ctx, "MkdirAll %s", filename)
+	defer func() {
+		fs.deferLog.CDebugf(fs.ctx, "MkdirAll done: %+v", err)
+	}()
+
+	parts := strings.Split(filename, "/")
```

Comment:
hmm I was under the impression that it's gonna be `"/"` no matter the platform, like https://golang.org/pkg/net/http/#FileSystem . But I can't seem to find relevant doc for `billy`